packages/ui-extensions-server-kit/src/types.ts

```diff
@@ -73,7 +73,7 @@ export interface ExtensionPayload {
   version: string
   surface: Surface
   title: string
-  extensionPoints?: string[]
+  extensionPoints?: string[] | {target: string; module: string; main: {url: string}}
```

Comment:
These types are duplicated with `packages/ui-extensions-server-kit/src/ExtensionServerClient/types.ts`.

I opened a WIP PR here to address the duplication and to unify the types: https://github.com/Shopify/cli/pull/833